lib/nostrum/struct/message/sticker.ex

```diff
@@ -0,0 +1,122 @@
+defmodule Nostrum.Struct.Message.Sticker do
+  @moduledoc """
+  Struct representing a Discord message.
```

Comment:
This docstring probably needs to be adjusted:

```suggestion
  Struct representing a sticker that can be sent in messages.
```